style.css

```diff
@@ -109,12 +123,252 @@ h2 {
   justify-content: left;
   align-items: center;
   gap: 1.25rem;
 }
 
 .social-media a :hover {
-  fill: #3909c8;
+  color: #3909c8;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+  grid-gap: 1.5rem;
+}
+
+.project-card {
+  padding: 0.75rem;
+  margin: 0.03 auto;
+  width: 100%;
+  max-width: 20.625rem;
+  border-radius: 0.05rem;
+}
+
+.project-title {
+  font-family: poppins, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  letter-spacing: 0.03em;
+}
+
+.programming-languages {
+  color: solid #000;
+  display: flex;
+  list-style-type: none;
+  padding: 0.05;
+  justify-content: left;
+  align-items: center;
+  gap: 1.25rem;
+}
+
+.counter {
+  display: flex;
+  list-style-type: none;
+  padding: 0.05;
+  justify-content: left;
+  align-items: center;
+  gap: 1.25rem;
+}
+
+.canopy-text {
+  display: flex;
+  padding: 0.05;
+  justify-content: left;
+  align-items: center;
+  gap: 1.25rem;
+  font-family: poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.canopy {
+  font-family: poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.info-section {
+  padding-left: 1.5rem;
+  padding-right: 1.5rem;
+}
+
+.about-info {
+  max-width: 37.5rem;
+  margin: 1.5rem auto;
+}
+
+.info-container {
+  max-width: 37.5rem;
+  margin: 1.5rem auto;
+}
+
+.primary-text {
+  font-family: poppins, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.btn {
+  color: #396df2;
+  background: #efebeb;
+}
+
+.see-project-button {
+  color: #396df2;
+  font-family: poppins, sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  border-radius: 25px;
+  border: 2px solid #396df2;
+  padding: 5px;
+  width: 200px;
+  height: 150px;
+}
+
+.list-frames {
+  list-style-type: none;
+  left: 12px;
+  top: 12px;
+  border-radius: 8px;
+}
+
+.language-framework {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 90rem;
+  margin: auto;
+  padding: 1rem;
+}
+
+.language-skills {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 90rem;
+  margin: auto;
+  padding: 1rem;
+}
+
+.framework {
+  font-family: poppins, sans-serif;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.skills {
+  font-family: poppins, sans-serif;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.05em;
+  text-align: left;
+}
+
+.headline {
+  background-color: rgb(75, 54, 230);
+  background-image: url(images/header-shapes\ mobile@2x.svg);
+  left: 0.05px;
+  top: 92px;
+  border-radius: 0.05px;
+  padding: 114px 24px 114px 24px;
+}
```

Comment:
- [ ] Check your background image, it's not the same as the FIGMA template. Since the colors of the forms are different

![image](https://user-images.githubusercontent.com/93630700/205455520-05113de7-7301-453b-b671-51824e20073c.png)

![image](https://user-images.githubusercontent.com/93630700/205455527-3412f90d-7bc6-48bb-be76-fe98d772bcc1.png)

- [ ] Kindly change the font color, so it looks equal to the template. White is the correct one! 💯 

![image](https://user-images.githubusercontent.com/93630700/205455554-d8141dfb-c096-4ceb-b1e3-2068f5468224.png)

- [ ] Kindly make the inputs field have the same styles as the FIGMA template

![image](https://user-images.githubusercontent.com/93630700/205455575-ca236987-0714-4e7c-bffe-23ed683069bf.png)

![image](https://user-images.githubusercontent.com/93630700/205455624-59ebdd8f-a0f9-49cc-bcd0-2180c6ff9265.png)

- [ ] At the bottom of the contact form, there is a black line missing. Please add it ❗ 

![image](https://user-images.githubusercontent.com/93630700/205455647-c1f344fb-bb65-408c-85d2-3d6353fc1108.png)